scripts/fuzz_opt.py

```diff
@@ -126,16 +126,33 @@ def no_pass_debug():
 def randomize_feature_opts():
     global FEATURE_OPTS
     FEATURE_OPTS = CONSTANT_FEATURE_OPTS[:]
-    # 1/3 the time apply all the possible opts, 1/3 none of them, to maximize
-    # coverage both ways, and 1/3 pick each one randomly
-    if random.random() < 0.33333:
+
+    # Note that POSSIBLE_FEATURE_OPTS contains --disable-X for each feature, so
+    # we enable all features when we add nothing from it, etc. (assert on that
+    # form here to guard against future refactorings making a mess here).
+    assert '--disable' in POSSIBLE_FEATURE_OPTS[0]
+    assert '--enable' not in POSSIBLE_FEATURE_OPTS[0]
```

Comment:
How about renaming it to `POSSIBLE_DISABLE_FEATURE_OPTS` or something? Code like
```py
FEATURES_OPTS += POSSIBLE_FEATURE_OPTS
```
looks like we are enabling all the possible options out there when we are actually doing the very opposite.

Reply:
Good idea, thanks, renamed and refactored.